charts/openshift-metering/templates/presto/presto-coordinator-config.yaml

```diff
@@ -59,6 +59,17 @@ data:
 {{- end }}
 
   jvm.config: |
+{{- if .Values.presto.spec.config.tls.enabled }}
+    -Dhttp.proxyHost=networking.useGlobalProxyNetworking.proxy.config.http_proxy.hostname
+    -Dhttp.proxyPort=networking.useGlobalProxyNetworking.proxy.config.http_proxy.port
+    -Dhttp.proxyUser=networking.useGlobalProxyNetworking.proxy.config.http_proxy.username
+    -Dhttp.proxyPassword=networking.useGlobalProxyNetworking.proxy.config.http_proxy.password
+{{ else }}
+    -Dhttps.proxyHost=networking.useGlobalProxyNetworking.proxy.config.https_proxy.hostname
+    -Dhttps.proxyPort=networking.useGlobalProxyNetworking.proxy.config.https_proxy.port
+    -Dhttps.proxyUser=networking.useGlobalProxyNetworking.proxy.config.https_proxy.username
+    -Dhttps.proxyPassword=networking.useGlobalProxyNetworking.proxy.config.https_proxy.password
```

Comment:
A couple of things to note about this current iteration of the jvm configuration:
1.) We only want to set these properties when a cluster-wide Proxy is enabled, so we do care about whether or not TLS is enabled for Presto, but in the overall context of the Proxy being enabled. In more concrete terms:
```
if proxy is enabled:
  if tls is enabled:
    set `-Dhttps.proxy*` configurations
  else:
    set `-Dhttp.proxy*` configurations
```
2.) Without templating (`{{ ... }}`), you're essentially just passing this string to the jvm configuration, and not injecting any values from values.yaml.